apps/web/components/ui/avatar/UserAvatar.tsx

```diff
@@ -0,0 +1,19 @@
+import { getUserAvatarUrl } from "@calcom/lib/getAvatarUrl";
+import type { User } from "@calcom/prisma/client";
+import { Avatar } from "@calcom/ui";
+
+type UserAvatarProps = Omit<React.ComponentProps<typeof Avatar>, "alt" | "imageSrc"> & {
+  user: Pick<User, "organizationId" | "name" | "username">;
+  /**
+   * Useful when allowing the user to upload their own avatar and showing the avatar before it's uploaded
+   */
+  imageSrc?: string | null;
```

Comment:
maybe we should rename this props to previewSrc or previewImgSrc to make it more clear that it is supposed to be used to display avatar while it's uploading, so that there is no mistake when using this component.

Reply:
Agreed. Couldn't find a better name, but this sounds better.

Reply:
Done